values/knative-serving/knative-serving-cr.gotmpl

```diff
@@ -1,12 +1,27 @@
 {{- $v := .Values }}
 {{- $k := $v.apps.knative -}}
+{{- $tag := "v1.18.1" -}}
+{{- $netIstioTag := "v1.18.0" -}}
 apiVersion: operator.knative.dev/v1beta1
 kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
 spec:
-  version: "1.15"
+  {{- if $v.otomi.linodeLkeImageRepository }}
+  # Use private repository configuration when ORCS is enabled
+  # https://knative.dev/docs/install/operator/configuring-serving-cr/#private-repository-and-private-secrets
+  registry:
+    default: "{{ $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/serving/cmd/${NAME}:{{ $tag }}"
+    override:
+      # Overriding paths for images not under "gcr.io/knative-releases/knative.dev/serving/cmd" otherwise the system
+      # will try to pull them from "mirrored.registry/knative-releases/knative.dev/serving/cmd" which does not exist.
+      migrate: "{{ $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate:{{ $tag }}"
+      cleanup: "{{ $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/serving/pkg/cleanup/cmd/cleanup:{{ $tag }}"
+      net-istio-controller/controller: "{{ $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/net-istio/cmd/controller:{{ $netIstioTag }}"
+      net-istio-controller/webhook: "{{ $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/net-istio/cmd/webhook:{{ $netIstioTag }}"
+  {{- end }}
+  version: "1.18"
```

Comment:
Maybe we can also move it up there as a variable to have it all in one place. Otherwise this can easily be forgotten on the following upgrade.